app/backend/src/couchers/errors.py

```diff
@@ -61,6 +61,7 @@
 INVALID_HOST_REQUEST_STATUS = "You can't set the host request status to that."
 INVALID_MESSAGE = "Invalid message."
 INVALID_NAME = "Name not supported."
+INVALID_PHONE = "Phone number not in E.164 format."
```

Comment:
No, don't display this error to the user.

Reply:
That would require specially handling only this error compared to the others on the frontend. 

I agree in theory, but until we switch to error codes, it will make the frontend code very messy

Reply:
I changed the message to "Phone number must be in international format without punctuation." But I'd expect the frontend to do some text processing before passing it to the backend so it could as well check the format itself :)

The phonenumbers Python library could do local-to-international conversion, so I assume there is one for javascript as well. :)